beetsplug/alternatives.py

```diff
@@ -43,6 +43,19 @@ def update(self, lib, options):
                             .format(e.args[0]))
         alt.update(create=options.create)
 
+    def list_tracks(self, lib, options):
+        if options.format is not None:
+            beets.config[beets.library.Item._format_config_key].set(
+                options.format)
```

Comment:
I think this should use `beets.ui._set_format`, see the usage in `beets.ui.add_format_option`. This should also fix the type errors on Python 2.

Reply:
This was my initial idea, too. However, `_set_format` is an internal method on the `CommonOptionsParser` object and I’d like to avoid constructing such an object if we don’t need it. I tackled the Python 2.7 error by using `decargs`.